include/real/real.hpp

```diff
@@ -59,12 +60,14 @@ namespace boost {
 
             KIND _kind;
 
+            std::variant<real_explicit, real_algorithm> diff_number;
```

Comment:
Why "diff_number", if it is not obvious I would put a comment about it.

Reply:
I agree with Damian, but I would find a better name rather than fixing it with a comment.